torchtext/transforms.py

```diff
@@ -221,6 +223,33 @@ def forward(self, input: Any) -> Any:
         return F.add_token(input, self.token, self.begin)
 
 
+class PadTransform(Module):
+    def __init__(self, max_length: int, pad_value: int):
+        super().__init__()
+        self.max_length = max_length
+        self.pad_value = pad_value
+
+    def forward(self, x: torch.Tensor) -> torch.Tensor:
+        max_encoded_length = x.size(-1)
+        if max_encoded_length < self.max_length:
+            pad_amount = self.max_length - max_encoded_length
+            x = torch.nn.functional.pad(x, (0, pad_amount), value=self.pad_value)
+        return x
```

Comment:
could you please add doc-strings for `__init__` and `forward` methods? Also update the the [rst file](https://github.com/pytorch/text/blob/main/docs/source/transforms.rst) to update the documentation.